example/lib/main.dart

```diff
@@ -252,6 +266,14 @@ class _HomePageState extends ConsumerState<HomePage> {
           }),
       body: Builder(builder: (context) {
         return NavbarRouter(
+          swipeable: swipeable,
+          // swipeableLeftArea: Rect.fromLTWH(
+          //     0, 50, 50, MediaQuery.of(context).size.height * 0.9),
+          // swipeableRightArea: Rect.fromLTWH(
+          //     MediaQuery.of(context).size.width - 50,
+          //     50,
+          //     50,
+          //     MediaQuery.of(context).size.height * 0.9),
```

Comment:
remove this comment and add

```suggestion
 swipeableArea: Rect.fromLTWH(0, 0, MediaQuery.of(context).size.width,
```

Even without specifying `swipeableArea` whole width should be swipeable